### ISSUE TITLE
Allow rotation of Wipe Tower

### ISSUE THREAD

**Issue by eschreiner** [SoftFever/OrcaSlicer#3596]
**Is your feature request related to a problem? Please describe.**
I have one large object to print where the prime tower only fits into a corner if rotated by 45°

**Which printers will be beneficial to this feature**
All with MMU

**Describe the solution you'd like**
The setting seems to be half way implemented as the search already knows about it, but the related controls are invisible (or I am too stupid). I am using a Bambu Lab P1S, but there seems to be no good reason as to why this setting should not be available here.

**Describe alternatives you've considered**
Hacking the 3MF file did not work :(

**Additional context**
see screenshot taken from Orca Slicer 1.9.0

![grafik](https://github.com/SoftFever/OrcaSlicer/assets/184471/eda840c8-4279-4dd4-bc8e-a2774bb03c81)

**Comment by streetgt** [comment]
+1 for this, would be great to have more options to control the Prime Tower location.

**Comment by apadua** [comment]
Agreed. Important feature.

**Comment by CristianGTR97** [comment]
THIS WOULD BE VERY USEFUL

**Comment by Gozania** [comment]
+1 for this... Bonus points if you just give us full control of the prime tower... (Placement wherever we want & positioned however we want.

**Comment by exsodus2** [comment]
I just ran into a print where this would have been very helpful. I had to cover calibration lines because I couldn't orient the tower. +1 for this feature

**Comment by Magic815** [comment]
Just ran into a print bed where being able to rotate the wipe tower would have been the difference between fitting all parts on one plate, versus having to spread them across two plate. +1 for this feature!

**Comment by Gozania** [comment]
Its truly asinine that full control of the prime tower hasn't been implemented yet....   Its obviously a feature that many of us want.

**Comment by ewilts** [comment]
+1

**Comment by mtbikemarc** [comment]
+1

**Comment by appleimperio** [comment]
+1

**Comment by wormiedk** [comment]
It seems to be in orca slicer 2.2.0 but for me it has no effect

<img width="402" alt="Image" src="https://github.com/user-attachments/assets/c75e3a28-3242-4867-9009-d8fe58f5ebae" />

**Comment by Jazun713** [comment]
doesn't seem to be working for 2.3.0-dev either.

**Comment by hugues974974** [comment]
Yes this problem is still there.... Prime tower can be turned in Prusa Slicer but doesn't work in ORCA.
It is very usefuk to give that prime tower an angle for large prints. 
Awaiting for a correction in a last version ???
Best regards,
Hugues

**Comment by Magic815** [comment]
Would still love to see this feature working

**Comment by appleimperio** [comment]
If I rotate the wipe tower in Orca Slicer 2.3.1 the application hangs. We need this fix

**Comment by Kathode-Ray** [comment]
Wipe tower rotation angle parameter doesn't do anything. I have set it to 45 degress, but it stays stuck at the shown position.

![Image](https://github.com/user-attachments/assets/8216be4c-7c46-4666-91d9-9f67b815e787)

**Comment by olli4711** [comment]
Hi,
Version 2.3.2-dev, Build 737948b
Wipe tower rotation angle is visible in Prime tower settings but has no effect.

+1 for the feature to be implemented - had a print where prime tower orientation towards case fans was critical and had to use CRY Print to rotate tower :-D

**Comment by Le-zOU** [comment]
still not fixed in 2.4.0 ?????

**Comment by Kathode-Ray** [comment]
Nope. I'm using the Orca-WaveOverhangs fork, which is base on 2.4.0-dev and the issue is still present.

<img width="1426" height="577" alt="Image" src="https://github.com/user-attachments/assets/39a44476-2317-4026-b354-53639d6703bd" />

**Comment by jimmy-brightz** [comment]
This still isn't working in V2.4.2. I'm working on a project where I would really love to be able to use this feature. The prime tower keeps getting in the way with its default rotation.
